include/Jit/LLILCJit.h

```diff
@@ -164,6 +167,35 @@ struct LLILCJitPerThreadState {
   std::map<CORINFO_FIELD_HANDLE, uint32_t> FieldIndexMap;
 };
 
+/// \brief Stub \p SymbolResolver that tells dynamic linker not to apply
+/// relocations for external symbols we know about.
+///
+/// The ObjectLinkingLayer takes a SymbolResolver ctor parameter.
+class EESymbolResolver : public llvm::RuntimeDyld::SymbolResolver {
+public:
+  EESymbolResolver(llvm::StringMap<uint64_t> *NameToHandleMap) {
+    this->NameToHandleMap = NameToHandleMap;
+  }
+
+  llvm::RuntimeDyld::SymbolInfo findSymbol(const std::string &Name) final {
+    // Address 0 means that we don't know how to resolve this symbol, which will
+    // result in an LLVM error.
```

Comment:
Why delay the error?  Seems more straightforward to assert count == 1 here and unconditionally return UINT64_MAX as the address.